View/media_control_panel.py

```diff
@@ -11,3 +11,17 @@ def __init__(self):
         to the panel using some layout.
         """
         super().__init__()
+        
+        # create play pause btn
+        self.playBtn = QPushButton()
+        self.playBtn.setIcon(self.style().standardIcon(QStyle.SP_MediaPlay))
+        #create horizontal box layout
+        hboxLayout = QHBoxLayout()
```

Comment:
Variable names should be snake_case. Instead of "hboxLayout" it could instead be "horizontal_layout".